src/studio/src/designer/frontend/ux-editor/containers/FormDesigner.tsx

```diff
@@ -129,156 +130,117 @@ const styles = ((theme: Theme) => createStyles({
-    return (
-      <DndProvider backend={HTML5Backend}>
-        <div className={classes.root}>
+  return (
+    <DndProvider backend={HTML5Backend}>
+      <div className={classes.root}>
+        <Grid
+          container={true}
+          wrap='nowrap'
+          spacing={0}
+          classes={{ container: classNames(classes.container) }}
+          id='formFillerGrid'
+        >
           <Grid
-            container={true}
-            wrap='nowrap'
-            spacing={0}
-            classes={{ container: classNames(classes.container) }}
-            id='formFillerGrid'
+            item={true} xs={2}
+            className={classes.toolbarWrapper} classes={{ item: classNames(classes.item) }}
           >
-            <Grid
-              item={true} xs={2}
-              className={classes.toolbarWrapper} classes={{ item: classNames(classes.item) }}
-            >
-              <Toolbar />
-            </Grid>
-            <Grid
-              item={true} xs={8}
-              className={classes.mainContent} classes={{ item: classNames(classes.item) }}
-            >
-              <div className={classes.versionControlHeaderMargin}>
-                <VersionControlHeader language={this.props.language} />
-              </div>
-              <div className={classes.pageHeader}>
-                <Typography classes={{ root: classes.pageHeaderText }}>
-                  {`Side - ${this.props.selectedLayout}`}
-                </Typography>
-              </div>
-              <div
-                style={{
-                  width: 'calc(100% - 48px)',
-                  paddingTop: '12px',
-                  marginLeft: '24px',
-                }}
-              >
-                <DesignView />
-                {this.state.codeEditorOpen ?
-                  this.renderLogicEditor()
-                  : null}
-              </div>
-            </Grid>
-            <Grid
-              item={true}
-              xs={2}
-              classes={{ item: classNames(classes.item) }}
+            <Toolbar />
+          </Grid>
+          <Grid
+            item={true} xs={8}
+            className={classes.mainContent} classes={{ item: classNames(classes.item) }}
+          >
+            <div className={classes.versionControlHeaderMargin}>
+              <VersionControlHeader language={language} />
+            </div>
+            <div className={classes.pageHeader}>
+              <Typography classes={{ root: classes.pageHeaderText }}>
+                {`Side - ${selectedLayout}`}
+              </Typography>
+            </div>
+            <div
+              style={{
+                width: 'calc(100% - 48px)',
+                paddingTop: '12px',
+                marginLeft: '24px',
+              }}
             >
-              <RightMenu
-                toggleFileEditor={this.toggleCodeEditor}
-                language={this.props.language}
-              />
-            </Grid>
+              <DesignView dispatch={null} />
```

Comment:
Should dispatch be passed as prop here?

Reply:
This is what we need to do if dispatch is not optional in the prop interface.